js/src/Components/NLPGenerator.tsx

```diff
@@ -115,6 +115,9 @@ interface SingleTextProp {
   modelName: string;
 }
 
+var Filter = require('bad-words'),
```

Comment:
```suggestion
import Filter from 'bad-words'
```

Use import semantics for consistency

Reply:
Done.